integrations/pooltogether-v4/utils/prize-pool-contract-list.ts

```diff
@@ -0,0 +1,5 @@
+export const PRIZE_POOL_CONTRACT = {
+  1: '0xd89a09084555a7D0ABe7B111b1f78DFEdDd638Be',
+  10: '0x79Bc8bD53244bC8a9C8c27509a2d573650A83373',
+  137: '0x19DE635fb3678D8B8154E37d8C9Cdf182Fe84E60',
+}
```

Comment:
If you declare an address list or object `as const` you'll get the proper typing instead of getting `string` 

```suggestion
} as const
```